runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsOverlay.java

```diff
@@ -51,10 +51,10 @@
 	private static final int FPS_SIZE = MAX_FPS + 1;
 	private static final int Y_OFFSET = 14;
 	private static final int VALUE_X_OFFSET = 15;
+    private static final String FPS_STRING = " FPS";
```

Comment:
Indentation

Reply:
Hmm, weird. I changed my IDE so it uses tabs instead of spaces because Travis failed, and now the indentation is messed up in Github. I'll look into it.

Reply:
Fixed